### PR TITLE
[4.0][codemirror] Add space between attributes

### PR DESCRIPTION
**Pull request by Quy**
### Summary of Changes
Fix markup. Add space between attributes.

### Testing Instructions
Go to System > Plugins.
Edit `Editor - CodeMirror`.
View page source.
Find `<joomla-editor-codemirror`.
See no space between attributes.

### Actual result
codemirror

### CODE REVIEW

File: plugins/editors/codemirror/layouts/editors/codemirror/element.php [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
 $basePath        = $params->get('basePath', 'media/vendor/codemirror/');
 $modePath        = $params->get('modePath', 'media/vendor/codemirror/mode/%N/%N');
-$modPath         = 'mod-path="' . Uri::root() . $modePath . $extJS . '"';
+$modPath         = ' mod-path="' . Uri::root() . $modePath . $extJS . '"';
 $fskeys          = $params->get('fullScreenMod', array());
 $fskeys[]        = $params->get('fullScreen', 'F10');
 $fullScreenCombo = implode('-', $fskeys);
-$fsCombo         = 'fs-combo=' . json_encode($fullScreenCombo);
-$option          = 'options=' . json_encode($options);
+$fsCombo         = ' fs-combo=' . json_encode($fullScreenCombo);
```
Comment (brianteeman) [inline plugins/editors/codemirror/layouts/editors/codemirror/element.php]: I usually get told off for doing things like that with a space at the beginning ;)
Could you not end the line with a \n

Comment (Quy) [reply plugins/editors/codemirror/layouts/editors/codemirror/element.php]: OK. How about reverting and adding directly here?

`<joomla-editor-codemirror <?php echo $editor . ' ' . $addons . ' ' . $modPath . ' ' . $fsCombo . ' ' . $option; ?>>`

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 9926f893d0f23e87a1e7650911160e53a3971350<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27225">issues.joomla.org/tracker/joomla-cms/27225</a>.</sub>

**Comment by uGE70** [comment]
I have tested this item :white_check_mark: successfully on 9926f893d0f23e87a1e7650911160e53a3971350<br /><br />Spaces added between attributes of tag joomla-editor-codemirror after patch applied<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27225">issues.joomla.org/tracker/joomla-cms/27225</a>.</sub>

**Comment by SharkyKZ** [comment]
RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27225">issues.joomla.org/tracker/joomla-cms/27225</a>.</sub>

File: plugins/editors/codemirror/layouts/editors/codemirror/element.php (hunk 1 above)
Comment (HLeithner) [reply plugins/editors/codemirror/layouts/editors/codemirror/element.php]: that would be better

**Pull request merged**